http-clients/src/main/java/org/triplea/http/client/lobby/game/remote/actions/RemoteActions.java

```diff
@@ -0,0 +1,29 @@
+package org.triplea.http.client.lobby.game.remote.actions;
+
+import java.net.InetAddress;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.Set;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.triplea.http.client.IpAddressParser;
+
+@ToString
+@Builder
+@EqualsAndHashCode
+public class RemoteActions {
+  private final Collection<BannedPlayer> bannedPlayers;
+  private final Collection<String> serversToShutdown;
+
+  public boolean serverIsRequestedToShutdown(final InetAddress myPublicIpAddress) {
+    return serversToShutdown != null
+        && serversToShutdown.stream()
+            .map(IpAddressParser::fromString)
+            .anyMatch(myPublicIpAddress::equals);
```

Comment:
Hmm it looks like most of the implementations of `serversToShutdown` are sets.
Would it work converting InetAddress to a string an performing a `serversToShutdown.contains(myPublicIpAddress.toString())`?
Alternatively converting serversToShutdown to a collection of InetAddresses upon creation?

Reply:
I'm afraid neither may work. For Strings, IPV6 can be represented in multiple formats, so string representations could be brittle. As `RemoteActions` can be sent as  JSON payload, I'd rather avoid using nested objects where we can.

Reply:
Bummer, but ok